client/src/App.tsx

```diff
@@ -0,0 +1,17 @@
+import swapGif from "/swap.gif";
+import "./App.css";
+
+function App() {
+  return (
+    <>
+      <div>
+        <a href="https://www.swap-commerce.com/" target="_blank">
+          <img src={swapGif} className="logo" alt="Vite logo" />
```

Comment:
Nitpick suggestion, just to align with the contents of the image:

```suggestion
          <img src={swapGif} className="logo" alt="Swap pigeon" />
```

Reply:
haha sure